# Object description container
temoto_2/ObjectContainer[] objects

---

temoto_2/RMPResponse rmp
